modules/shared/system_module_version.mk

```diff
@@ -21,6 +21,11 @@ PREBOOTLOADER_MBR_VERSION ?= 2
 PREBOOTLOADER_PART1_VERSION ?= 3
 endif
 
+ifeq ($(PLATFORM_ID),28)
```

Comment:
No need as of right now for separate dependencies. Use `PLATFORM_MCU` == rtl872x or both trackerm and p2 for checks.